subprojects/plugins/src/main/java/org/gradle/api/plugins/internal/SourceSetUtil.java

```diff
@@ -74,7 +74,7 @@ public Object call() {
     }
 
     public static void configureOutputDirectoryForSourceSet(final SourceSet sourceSet, final SourceDirectorySet sourceDirectorySet, final Project target,
-            Provider<? extends AbstractCompile> compileTask, Provider<CompileOptions> options) {
+           Provider<? extends AbstractCompile> compileTask, Provider<CompileOptions> options) {
```

Comment:
Either put every argument on a single line or put everything in one line, but don't split the argument list in the middle.